cmd/validatorcmd/increaseBalance.go

```diff
@@ -64,54 +65,18 @@ func increaseBalance(_ *cobra.Command, _ []string) error {
 	if err != nil {
 		return err
 	}
-	var balance uint64
-	var validationID ids.ID
-	if validationIDStr != "" {
-		validationID, err = ids.FromString(validationIDStr)
-		if err != nil {
-			return err
-		}
-	} else {
-		isBootstrapValidator, err := app.Prompt.CaptureYesNo("Is the validator a bootstrap validator?")
-		if err != nil {
-			return err
-		}
-		if isBootstrapValidator {
-			if l1 == "" {
-				return fmt.Errorf("--l1 flag is required to get bootstrap validator balance")
-			}
-			sc, err := app.LoadSidecar(l1)
-			if err != nil {
-				return fmt.Errorf("failed to load sidecar: %w", err)
-			}
-			if !sc.Sovereign {
-				return fmt.Errorf("avalanche validator increaseBalance command is only applicable to sovereign L1s")
-			}
-			bootstrapValidators := sc.Networks[network.Name()].BootstrapValidators
-			if len(bootstrapValidators) == 0 {
-				return fmt.Errorf("this L1 does not have any bootstrap validators")
-			}
-			bootstrapValidatorsString := []string{}
-			bootstrapValidatorsToIndexMap := make(map[string]int)
-			for index, validator := range bootstrapValidators {
-				bootstrapValidatorsString = append(bootstrapValidatorsString, validator.NodeID)
-				bootstrapValidatorsToIndexMap[validator.NodeID] = index
-			}
-			chosenValidator, err := app.Prompt.CaptureList("Which bootstrap validator do you want to get balance of?", bootstrapValidatorsString)
-			if err != nil {
-				return err
-			}
-			validationID, err = ids.FromString(bootstrapValidators[bootstrapValidatorsToIndexMap[chosenValidator]].ValidationID)
-			if err != nil {
-				return err
-			}
-		} else {
-			validationID, err = app.Prompt.CaptureID("What is the validator's validationID?")
-			if err != nil {
-				return err
-			}
-		}
+
+	validationID, cancel, err := getNodeValidationID(network, l1, nodeIDStr, validationIDStr)
+	if err != nil {
+		return err
+	}
+	if cancel {
+		return nil
 	}
+	if validationID == ids.Empty {
+		return fmt.Errorf("the specified node is not a L1 validator")
```

Comment:
nit: add explanation to the error msg like `the specified node is not a L1 validator as validationID is empty` 

pls feel free to ignore this

Reply:
ok this will ignore